### PR TITLE
CE-216: Refresh Email Signup Footer (ESB)

### PR DESCRIPTION
**Pull request by mrfantasticwonders**
**Purpose**
> As a site visitor, when I scroll to the bottom of any page, I want to see a newsletter subscription bar rise up from the footer to catch my eye and prompt me to sign up for some tasty content.

This PR adds a new button to be used for signup forms called `signup-button`.

**JIRA**
https://careerbuilder.atlassian.net/browse/CE-216

**Changes**
* Improvements and fixes
  * New `signup-button`

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * N/A

* Migrations or Steps to Take on Production
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
screen shot 2017-10-17 at 5 10 47 pm

screen shot 2017-10-17 at 5 11 01 pm

* After
screen shot 2017-10-17 at 5 11 13 pm

screen shot 2017-10-17 at 5 11 22 pm

**Feature Server**
http://web.employer-1.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * All pages

* Steps to take
  * Go to any page on the site and scroll to the bottom to see the new signup form
  * Compare it to the mocks for both mobile and desktop
  * View it on your phone to see if it looks ok
  * Resize your browser
  * Compare the button to the hiringDot site. It should behave similarly when resizing the browser.

* Responsive considerations
  * Resize your browser and test it on your phone. The three column layout should collapse to a stacked view around 750px.

**Relevant PRs/Dependencies**
  * https://github.com/cbdr/employer/pull/1016

**Additional Information**
N/A

### CODE REVIEW

File: sass/directives/02_base_components/button/_button.scss [hunk 1]
```diff
@@ -193,6 +193,27 @@ $massive-button-horizontal-padding: $base-input-font-size * 2;
   }
 }
 
+@mixin signup-button {
```
Comment (toastercup) [inline sass/directives/02_base_components/button/_button.scss]: I feel that this naming may lock us in to its use case. We have examples of this happening across Employer that we still haven't been able to unravel, and I feel button styles are very frequently re-used. Should we go with the prior convention of just giving buttons numbered names?

Comment (mrfantasticwonders) [reply sass/directives/02_base_components/button/_button.scss]: @toastercup I'm fine with changing this. I'd go with `senary-button` for this. It's not being used anywhere else in the site, so placing it at 6 would be fine. But the buttons are tiered in a way...

Comment (toastercup) [reply sass/directives/02_base_components/button/_button.scss]: :+1:

**Comment by mrfantasticwonders** [comment]
@toastercup This is ready for your review again!

**Comment by toastercup** [comment]
@mrfantasticwonders should this new button get added to the pattern guide?

**Comment by mrfantasticwonders** [comment]
@toastercup Yes! I added the necessary files to make this happen. I'm going to merge this branch with master and run a few commands to update the production build of Fractal.

**Pull request merged**